### PR TITLE
ledger-api-test-tool: Enable `MultiPartySubmissionIT` for everyone. [KVL-1261]

### PR DESCRIPTION
**Pull request by SamirTalwar-DA**
### Changelog

- **[Ledger API Test Tool]** The `MultiPartySubmissionIT` test case has been enabled by default. You will most likely not need to change anything.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/ledger-api-test-tool-on-canton/BUILD.bazel [hunk 1]
```diff
@@ -85,6 +85,8 @@ conformance_test(
             "CommandDeduplicationIT:SimpleDeduplicationBasic",  # sync vs async error (part of canton #6301)
             "CommandDeduplicationIT:DeduplicateSubmitterBasic",
             "CommandDeduplicationIT:DeduplicateUsingOffsets",  # temporary disabled until we adapt it for canton
+            "MultiPartySubmissionIT:MPSLookupOtherByKeyInvisible",  # Enable once error parsing more permissive
+            "MultiPartySubmissionIT:MPSCreateInsufficientAuthorization",  # canton returns INTERNAL instead of INVALID_ARGUMENT
```
Comment (nicu-da) [inline ledger/ledger-api-test-tool-on-canton/BUILD.bazel:89]: Is that expected? Is the canton team aware of this?

**Review by nicu-da — APPROVED** [review]

**Review by fabiotudone-da — APPROVED** [review]

File: ledger/ledger-api-test-tool-on-canton/BUILD.bazel (hunk 1 above)
Comment (SamirTalwar-DA) [reply ledger/ledger-api-test-tool-on-canton/BUILD.bazel:89]: No idea but I just moved them upwards.

@oliverse-da, are you aware we're ignoring these test cases?

Comment (oliverse-da) [reply ledger/ledger-api-test-tool-on-canton/BUILD.bazel:89]: Was aware of the first one failing, but not the second one. 👀

Comment (SamirTalwar-DA) [reply ledger/ledger-api-test-tool-on-canton/BUILD.bazel]: Both pass on my machine so I'm enabling them both. If they pass on CI, win.

**Review by mziolekda — APPROVED** [review]

**Pull request merged**